plugins/callback/foreman.py

```diff
@@ -98,12 +98,15 @@ def build_log(data):
     config_report API.
     """
     for source, msg in data:
-        if 'failed' in msg:
+        if msg.get('failed') or msg.get('stderr_lines') or (msg.get('msg') and 'Err' in msg.get('msg')):
```

Comment:
- the stderr_lines can exist and not mean failure IMO :thinking: take wget, it writes in stderr just because stdout is for the download result.
- the last bit seems quite arbitrary and could be possibly find even in success.
- could we extract it in a method? It feels already quite complicated

Reply:
I assume ansible must somehow keep the status in the task object represented, it's stdout also shows green/red based on the status. Could it be found somewhere in the `source`? Or do the logs from `data` have more information?

Reply:
@ares 
I think there is no details about the task in the source or data but I will check again to make sure.
@ezr-ondrej 
I agree but there are errors that I managed to catch only this way